Makefile

```diff
@@ -74,7 +74,7 @@ test: lint pythoncheck
 	python examples/pond-simple.py
 	python examples/securenn-playground.py
 	python examples/federated-average/run.py
-	python -m unittest discover
+	python -m unittest discover -f
```

Comment:
should this be default on CI?

Reply:
In my opinion yes! Early fail if anything goes wrong so you don't have to wait the full test run to find out your build has failed.

Reply:
alright!